viewer/src/locales/Spanish.js

```diff
@@ -243,25 +243,25 @@ export default {
 
     // Menu
     "Menu":
-        null,
+        "Menú",
     "Document":
-        null,
+        "Documento",
     "About":
-        null,
+        "Acerca",
     "Print":
-        null,
+        "Imprimir",
     "Close":
-        null,
+        "Cerrar",
     "View mode":
-        null,
+        "Modo de visualización",
     "Scale":
-        null,
+        "Escala",
     "Rotation":
-        null,
+        "Rotación",
     "Cursor mode":
-        null,
+        "Modo del cursor",
     "Full page mode":
-        null,
+        "Modo de página completa",
     "Fullscreen mode":
-        null,
-};
\ No newline at end of file
+        "Modo de oantalla completa",
```

Comment:
@osvaldozoso Probably a typo. I will change it to `pantalla`, if you don't mind.